### ISSUE TITLE
Request: festivals.json to populate team result isHoragaiRatioTop during festival?

### ISSUE THREAD

**Issue by Bluebie** [misenhower/splatoon3.ink#61]
Unsure if this is possible, as this information doesn't appear in the mobile app, and I haven't dug deep in to how splatoon3.ink scrapes it's data, but saw some code about NSO integration that made me wonder if splat3.ink is accessing the same APIs the actual game uses for some things?

![string of fairy lights shaped like stars, glowing yellow, blue, and red, the same colors as the splatfest teams in the current splatoon 3 festival](https://github.com/misenhower/splatoon3.ink/assets/12768/c0e46ae9-aad2-4352-b30c-fa172864cd09)

During a splatfest when the festival officially begins, the idols announce which team got the most conch shells (horagai) in the lead up to the festival officially beginning. Half way thru fes, there's also the half time winner (not sure what this would be called) which affects tricolor match config. If it were possible to scrape this data and include it in festivals.json, it would be minorly nifty for cute home automation color scheme purposes.

Probably not worth the trouble, but i thought i'd ask just in case it's something not too tricky to access.

Thanks about it! Happy Festival!

And thanks so much for everything you've done already! 🥰

**Comment by Adrthegamedev** [comment]
> saw some code about NSO integration that made me wonder if splat3.ink is accessing the same APIs the actual game uses

I don't think so; reminder that Splatnet 3 needs NSO to work in the first place, and is part of the "**Nintendo Switch Online**" mobile app.
